src/Transactions/Transfer/TransferAddModalForm.js

```diff
@@ -21,7 +25,45 @@ import {
 
 const TRANSFER_FORM = 'transferForm';
 
-const TransferAddModalForm = ({ handleSubmit, onClose, funds }) => {
+const TransferAddModalForm = ({ fundId, handleSubmit, onClose, funds, store, dispatch, change }) => {
+  const formValues = getFormValues(TRANSFER_FORM)(store.getState()) || {};
+  const transferFrom = get(formValues, 'fromFundId');
+  const transferTo = get(formValues, 'toFundId');
+  const hasToFundIdProperty = Object.prototype.hasOwnProperty.call(formValues, 'toFundId');
```

Comment:
why not `formValues.hasOwnProperty('toFundId');` ?

Reply:
ESLint rule: Disallow use of Object.prototypes builtins directly (no-prototype-builtins)
https://eslint.org/docs/rules/no-prototype-builtins

Reply:
so why not `'toFundId' in formValues` ?

Reply:
Done